### DISCUSSION TITLE
Climate entity works but shows 'Unknown' mode/status when on (cooling)

### DISCUSSION THREAD

**Discussion by larryb31** [make-all/tuya-local#2065] [Q&A]
Hi, my climate entity works but displays 'Unknown' instead of Cool or Cooling. 
![image](https://github.com/make-all/tuya-local/assets/7194847/b4867b84-40b0-4198-b16a-df20b3c5b40b)

When clicking on the Mode, it displays 'Heat/Cool' instead of Cool. Is there something I can edit in the Climate.py file? 
State attributes for mode are:
hvac_modes:
  - "off"
  - cool
  - heat
  - dry
  - fan_only
  - heat_cool

API from Tuya for this is either:
{
        "code": "mode",
        "custom_name": "",
        "dp_id": 4,
        "time": 1718529322690,
        "type": "enum",
        "value": "cooling"
      },
or

{
        "code": "odu_mode",
        "custom_name": "",
        "dp_id": 107,
        "time": 1719549845057,
        "type": "enum",
        "value": "Cool"
      },

Appreciate any help, thanks.

**Comment by make-all** [comment]
It seems like your current temperature is also off by a factor of 10, are you sure you selected the correct config for your model?

**Reply by larryb31** [reply]
Hi, yes you are correct about being off by a factor of 10 but that's a separate issue, less critical for now. What do you mean by correct config for my model? I don't think my model is officially in the list of supported climate devices in the Tuya Local docs, however, I thought it may work with some manual tweaking somewhere.
